edge-functions/json-response/middleware.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @next/next/no-server-import-in-page */
+import { NextResponse, NextRequest } from 'next/server'
+
+export const config = {
+  matcher: '',
```

Comment:
Shouldn't this be '/file' without doing the check below?

Reply:
the logic should be to run on all pages apart from the file so as to filter downstream. It should be actually `matcher: '/'`. Cheers!